.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
```

Comment:
Looks great! Thank you very much for this. 
I was only wondering if this `master` branch is correct as we have `main` branch?

Reply:
I already fixed it without noticing Your commit, but You're right! Good catch 🧤

Reply:
```suggestion
      - main
```